<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1401</width>
    <height>901</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="graphic_roll_call_tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #282c34;

border: 3px solid #2f333d;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QFrame" name="mode_select_layout">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="mode_layout">
        <item>
         <widget class="QPushButton" name="check_mode">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
          </property>
          <property name="text">
           <string>Q 已到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="late_mode">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
          </property>
          <property name="text">
           <string>W 遲到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="missing_mode">
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
          </property>
          <property name="text">
           <string>E 曠課</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="normal_seat_layout">
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="information_tab">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>701</width>
      <height>900</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #21252b;

border: 3px solid #2f333d;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QFrame" name="horizontalFrame">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="info_copy_layout">
        <item>
         <widget class="QFrame" name="infos_btns_layout">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="infos_layout">
             <item>
              <widget class="QLabel" name="target_count">
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
               </property>
               <property name="text">
                <string>應到人數：0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="actual_count">
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
               </property>
               <property name="text">
                <string>實到人數：0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="missing_count">
               <property name="maximumSize">
                <size>
                 <width>130</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
               </property>
               <property name="text">
                <string>未到人數：0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="btns_layout">
             <item>
              <widget class="QPushButton" name="confirm_output">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
               </property>
               <property name="text">
                <string>確認</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clear_output">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
               </property>
               <property name="text">
                <string>清空</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="datetime_btn">
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>40</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>微軟正黑體</family>
                 <pointsize>12</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff;
background-color: #323842;

border-radius: 8px</string>
               </property>
               <property name="text">
                <string>選擇日期</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="copy_edit">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="student_table">
       <property name="styleSheet">
        <string notr="true">background-color: #323842;
border-radius: 5px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
